tests/Etc/Post.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Stancl\Tenancy\Tests\Etc;
+
+use Illuminate\Database\Eloquent\Model;
+use Stancl\Tenancy\Database\Concerns\BelongsToTenant;
+
+class Post extends Model
```

Comment:
We already have the Post & Comment models in https://github.com/tenancy-for-laravel/v4/pull/3/files.

Are these models different in a way that's *specific to RLS*? If yes, they should be in some more specific folder than just `tests/Etc`. If not, then add a note to change this once the PR with early identification changes is merged (use the same model in tests of both of the features).

Reply:
> We already have the Post & Comment models in https://github.com/tenancy-for-laravel/v4/pull/3/files.

We _had_ these – I moved these to the separate Post & Comment files

> Are these models different in a way that's specific to RLS? 

The models aren't different in that regard. Some are intended to be used with the single DB approach – Post uses BelongsToTenant and ScopedComment uses BelongsToPrimaryModel, so they're specific to the single DB approach. The Comment model isn't specific to any approach though. So that's a thing I'll need to make clear

> add a note to change this once the PR with early identification changes is merged (use the same model in tests of both of the features)

I'll add a note to sync the changes – in the early id PR, we have the separate Post & Comment files, but the ScopedX classes are defined only directly in the SingleDatabaseTenancyTest file. So in the early id PR, we'll need to use the separate ScopedX files.